src/Pages/HomePage/HomePage.css

```diff
@@ -0,0 +1,68 @@
+.homePage {
+  display: flex;
+  flex-direction: column;
+}
+.under {
+  background: #fff;
+}
+.searchBar {
+  margin-top: 250px;
```

Comment:
this is not good, it's a big value try to solve it without using this big value, maybe use flex